.github/workflows/pull_request.yml

```diff
@@ -1,10 +1,8 @@
 name: Pull request checks
 
 on:
-  push:
```

Comment:
no need to have that on 'push' events: we already run that on 'push' from the other workflow file, `push_image.yaml`